hack/test_local_build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
No one is going to be able to run this except `jamesh` due to push to `quay.io/jamesh` restricted. You may want to parametrize it to make it clearer